api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java

```diff
@@ -39,7 +39,7 @@
  *
  *   // It is recommended that the API user keep references to a Bound Counters.
  *   private static final BoundDoubleCounter someWorkBound =
- *       counter.bind(Collections.singletonList("SomeWork"));
```

Comment:
interesting. this shouldn't have even been valid. Are we missing some validation on the label-pairs used for binding?

Reply:
This is javadoc :)

Reply:
I don't think we test/run/validate javadocs

Reply:
oh lol.  ;)